cmd/worker/main.go

```diff
@@ -185,11 +212,16 @@ func main() {
 		log.Label("package_bucket", packagesBucket),
 		log.Label("results_bucket", resultsBucket),
 		log.Label("file_write_results_bucket", fileWritesBucket),
-		log.Label("image_tag", imageTag))
-
+		log.Label("image_tag", imageTag),
+		log.Label("topic_notification", topicNotificationURL))
+	
 	for {
-		err := messageLoop(ctx, subURL, packagesBucket, resultsBucket, fileWritesBucket, imageTag)
-		if err != nil {
+		topicNotification, err := pubsub.OpenTopic(ctx, topicNotificationURL)
```

Comment:
s/topicNotification/notificationTopic/ to be consistent with other vars like "resultsBucket", "imageTag"